src/types.ts

```diff
@@ -1,8 +1,9 @@
-export type EVENT_TYPE = "point_increase" | "user_share"
-export const EVENT_POINT_INCREASE: EVENT_TYPE = "point_increase"
-export const EVENT_USER_SHARE: EVENT_TYPE = "user_share"
+export type EVENT_TYPE = "point_increase" | "user_share" | "init_snapshot";
```

Comment:
init snapshot is not used
```suggestion
export type EVENT_TYPE = "point_increase" | "user_share";
```